rust/processor/src/processors/token_v2_processor.rs

```diff
@@ -1198,6 +1296,20 @@ async fn parse_v2_token(
             ))
     });
     current_token_royalties_v1.sort();
+    all_current_token_claims.sort_by(|a, b| {
```

Comment:
You can implement the Ord and PartialOrd trait for this struct. Example: https://github.com/aptos-labs/aptos-indexer-processors/blob/98386c6a27665f6ba4e7e7caf233e3f4581f5695/rust/processor/src/db/common/models/token_v2_models/v1_token_royalty.rs#L30